features/step_definitions/authentication_steps.rb

```diff
@@ -0,0 +1,27 @@
+Given(/^I have an account on the site$/) do
+  FactoryGirl.create(:user, email: "user@example.com", password: "password")
+end
+
+Given(/^My account is confirmed$/) do
+  User.last.confirm
```

Comment:
As a side note, I'm not a huge fan of this strategy and prefer the more generalized steps, but with the steps written the way they are here it seems like it would be useful.